internal/interfaces/grpc/permissions/permissions.go

```diff
@@ -300,6 +301,10 @@ func Whitelist() map[string][]bakery.Op {
 			Entity: EntityTrade,
 			Action: "write",
 		}},
+		"/grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo": {{
+			Entity: Reflection,
+			Action: "write",
+		}},
```

Comment:
do we really need this in whitelist? if yes, is there a way to NOT completely hardcode the endpoint like done in other methods?

Reply:
yeah, its needed
Done